src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes
+ * a string with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const firstArr = sourceString.split(/[;:]/);
+  const secondArr = firstArr.map(x => x.trim());
+  const result = {};
+
+  for (let i = 0; i < secondArr.length; i += 2) {
```

Comment:
Try to use `forEach or reduce` instead of `for`.

Also you could do trim inside `forEach`.